src/constants.h

```diff
@@ -119,7 +119,7 @@ PRODBETAEXPR(const char*, addonPublicKeyFile,
              ":/addons_signature/production.der",
              ":/addons_signature/staging.der");
 
-PRODBETAEXPR(qint64, keyRegeneratorTimeSec, 604800, 300);
+PRODBETAEXPR(qint64, keyRegeneratorTimeSec, 604800, 30);
```

Comment:
Was this intentional? I still think that 30 seconds is too fast.

Reply:
no it was not. It was me testing the bug.